integration/specs/DateTimePicker/datetimepicker-1.spec.js

```diff
@@ -0,0 +1,22 @@
+const PageDateTimePicker = require('../../../src/components/DateTimePicker/pageObject');
+
+const DATETIMEPICKER = '#datetimepicker-1';
+
+describe('DateTimePicker', () => {
+    beforeAll(() => {
+        browser.url('/#!/DateTimePicker/1');
+    });
+    beforeEach(() => {
+        browser.refresh();
+        const component = $(DATETIMEPICKER);
+        component.waitForExist();
+    });
+    it('should set correct date on modal header when day is clicked', () => {
+        const picker = new PageDateTimePicker(DATETIMEPICKER);
+        picker.click();
+        picker.waitUntilModalIsOpen();
```

Comment:
let's be consistent like we did in TimePicker page object so call this method waitUntilOpen